doc/man/8/mount.ceph.rst

```diff
@@ -23,170 +26,193 @@ client component does most of the real work. In fact, it is possible
 
-Each monitor address monaddr takes the form host[:port]. If the port
-is not specified, the Ceph default of 6789 is assumed.
+The first argument is the device part of the mount command. It includes host's
+socket and path within CephFS that will be mounted at the mount point. The
+socket, obviously, takes the form ip_address[:port]. If the port is not
+specified, the Ceph default of 6789 is assumed.
 
-Multiple monitor addresses can be separated by commas. Only one
-responsible monitor is needed to successfully mount; the client will
-learn about all monitors from any responsive monitor. However, it is a
-good idea to specify more than one in case one happens to be down at
-the time of mount.
+Multiple monitor addresses can be passed by separating them by commas. Only
+one monitor is needed to successfully mount; the client will learn about all
+monitors from any responsive monitor. However, it is a good idea to specify
+more than one in case the one happens to be down at the time of mount.
 
 If the host portion of the device is left blank, then **mount.ceph** will
 attempt to determine monitor addresses using local configuration files
 and/or DNS SRV records.
 
-A subdirectory subdir may be specified if a subset of the file system
-is to be mounted.
+A subset of the file system can be mounted by specifying the (absolute) path
+to the sub-directory right after ":" after the sockets.
 
-Mount helper application conventions dictate that the first two
-options are device to be mounted and destination path. Options must be
+Mount helper application conventions dictate that the first two options are
+device to be mounted and the mountpoint for that device. Options must be
 passed only after these fixed arguments.
 
 
 Options
 =======
 
-:command:`wsize`
-  int (bytes), max write size. Default: 16777216 (16*1024*1024) (writeback uses smaller of wsize
-  and stripe unit)
+:command:`cap_release_safety`
+    int, Default: calculated
 
-:command:`rsize`
-  int (bytes), max read size. Default: 16777216 (16*1024*1024)
+:command:`caps_wanted_delay_max`
+    int, cap release delay, Default: 60
 
-:command:`rasize`
-  int (bytes), max readahead. Default: 8388608 (8192*1024)
+:command:`caps_wanted_delay_min`
+    int, cap release delay, Default: 5
 
-:command:`osdtimeout`
-  int (seconds), Default: 60
+:command:`conf`
+    Path to a ceph.conf file. This is used to initialize the ceph context
+    for autodiscovery of monitor addresses and auth secrets. The default is
+    to use the standard search path for ceph.conf files.
 
-:command:`osdkeepalive`
-  int, Default: 5
+:command:`dirstat`
+    funky `cat dirname` for stats, Default: off
 
-:command:`mount_timeout`
-  int (seconds), Default: 60
+:command:`ip`
+    my ip
 
-:command:`osd_idle_ttl`
-  int (seconds), Default: 60
+:command: `mds_namespace=<fs-name>`
+      Specify the non-default file system to be mounted. Not passing this
+      option mounts the default file system.
```

Comment:
Let's put this at the top; it's probably the most likely option of interest to users.